.github/workflows/test-go.yml

```diff
@@ -24,4 +28,8 @@ jobs:
       - run: make test
       - uses: shogo82148/actions-goveralls@v1
         with:
-          path-to-profile: cover.out
\ No newline at end of file
+          path-to-profile: cover.out
+      - name: Docker Integration test
+        run:  make docker-integration-test
+      - name: Podman Integration test
+        run:  make podman-integration-test
```

Comment:
I would create different jobs for docker and podman testing

Reply:
Done